### ISSUE TITLE
Push only for Nextcloud?

### ISSUE THREAD

**Issue by Offerel** [bitfireAT/webdav-push#14]
Is this Push implementation only meant to work with Nextcloud? Personally, I'm not the biggest fan of Nextcloud, but I'm using SabreDAV with some patches. I'm very interested to have this also working with standard SabreDAV. I had a discussion with evert some years ago about the same topic, but we came to no real starting point.

Hopefully this will change now.

**Comment by rfc2822** [comment]
No, Push is intended as an open standard that can be used with every server. We only plan to make it working with Nextcloud as kind of a sample implementation.
